oozie-to-airflow/run-sys-test

```diff
@@ -187,6 +213,9 @@ Optional commands to execute:
 -A, --setup-autocomplete
         Sets up autocomplete for run-sys-tests
 
+-X, --setup-configuration
```

Comment:
I think we do not need that option at all. We should asks for each value individually if it is missing rather than asking to confirm whether configuration is needed.